guides/idea_setup.md

```diff
@@ -0,0 +1,36 @@
+# IntelliJ IDEA Project Setup Guide
+
+Follow these detailed steps to set up your project in IntelliJ IDEA seamlessly:
+
+1. Import Project from GitHub:
+    - Begin by importing the project from GitHub into IntelliJ.
+
+2. Build with Maven. Run the following commands from the command line:
+    - `mvn clean install -f dhis-2/pom.xml`
+    - `mvn clean install -f dhis-2/dhis-web/pom.xml`
+
+3. Add as Maven Project:
+    - Right-click on the project's pom.xml and select "Add as Maven Project".
+
+4. Configure Maven Projects:
+    - In the plugin list on the right side, click on Maven.
+    - Add the web project to the list of Maven projects.
+
+5. Configure Local Tomcat:
+    - Open the configurations menu and add a new local Tomcat configuration.
+    - In the deployment tab, add the artifact named _dhis-web-portal:war exploded_.
+![](resources/images/deploy.png)

```

Comment:
Make sure you mention that you do this but also, in the `Server` tab, you actually build the artifact before launch as we can see in the next picture.

Reply:
Adding the artifact to the list of artifacts to be deployed at startup, automatically adds it to the list of the ones to be built.